nixos-modules/host/options.nix

```diff
@@ -10,6 +10,11 @@
       '';
     };
 
+    host.cloud-hypervisor.enable = lib.mkEnableOption "Install a cloud-hypervisor security wrapper" // {
+      default = true;
+    };
+    host.cloud-hypervisor.package = lib.mkPackageOption pkgs "cloud-hypervisor" { };
```

Comment:
In the hindsight, it might be sufficient to just expose this one option. That'd have allowed me to implement the wrapper locally and configure `microvm.nix` to use it instead of the real thing. Even now I could do it it using an overlay, but re-evaluating Nixpkgs is a bit heavy handed for the task